experimental/material_3_demo/lib/component_screen.dart

```diff
@@ -691,52 +690,49 @@ class _ProgressIndicatorsState extends State<ProgressIndicators> {
         Row(
           children: [
             IconButton(
-              isSelected: playCircularProgressIndicator,
+              isSelected: playProgressIndicator,
               selectedIcon: const Icon(Icons.pause),
               icon: const Icon(Icons.play_arrow),
               onPressed: () {
                 setState(() {
-                  playCircularProgressIndicator =
-                      !playCircularProgressIndicator;
+                  playProgressIndicator = !playProgressIndicator;
                 });
               },
             ),
-            Expanded(
-              flex: 10,
-              child: Center(
-                child: playCircularProgressIndicator
-                    ? const CircularProgressIndicator(
-                        value: null,
-                      )
-                    : const CircularProgressIndicator(
-                        value: 0.7,
-                      ),
-              ),
-            ),
-            const Spacer(),
-          ],
-        ),
-        Row(
-          children: [
-            IconButton(
-              isSelected: playLinearProgressIndicator,
-              selectedIcon: const Icon(Icons.pause),
-              icon: const Icon(Icons.play_arrow),
-              onPressed: () {
-                setState(() {
-                  playLinearProgressIndicator = !playLinearProgressIndicator;
-                });
-              },
-            ),
-            Expanded(
-              child: playLinearProgressIndicator
-                  ? const LinearProgressIndicator(
-                      value: null,
-                    )
-                  : const LinearProgressIndicator(
-                      value: 0.7,
+            playProgressIndicator
+                ? Expanded(
+                    child: Row(
+                      children: const <Widget>[
+                        CircularProgressIndicator(
+                          value: null,
+                        ),
+                        SizedBox(
+                          width: 10,
+                        ),
+                        Expanded(
+                          child: LinearProgressIndicator(
+                            value: null,
+                          ),
+                        )
+                      ],
```

Comment:
Not to nitpick this to death, but these two sections look like they could be consolidated to something like:

```dart
final double? progressValue = playProgressIndicator ? null : 0.7;
```

And then just one: 

```dart
              Expanded(
                  child: Row(
                    children: const <Widget>[
                      CircularProgressIndicator(value: progressValue),
                      SizedBox(width: 10),
                      Expanded(
                        child: LinearProgressIndicator(value: progressValue),
                      )
```

Reply:
Oh, sorry about that! Fixed it!